dotcom-rendering/src/model/enhance-newsletters-page.ts

```diff
@@ -0,0 +1,93 @@
+import type { Newsletter } from '../types/content';
+import type {
+	DCRNewslettersPageType,
+	FENewslettersPageType,
+	GroupedNewsletters,
+} from '../types/newslettersPage';
+import STATIC_GROUP_DATA from './newsletter-grouping.json';
```

Comment:
Could `newsletter-grouping.json` possibly be turned into a Typescript file that exports an object instead? That way we get type safety and suggestions when editing that data.

Reply:
👍  done - I wasn't sure if it was best to consider the file to be configuration rather than code, but since its imported locally, probably best to use a Typescript file.